core/libp2p/transport/impl/upgrader_impl.cpp

```diff
@@ -10,8 +10,10 @@
 OUTCOME_CPP_DEFINE_CATEGORY(libp2p::transport, UpgraderImpl::Error, e) {
   using E = libp2p::transport::UpgraderImpl::Error;
   switch (e) {
-    case E::INTERNAL_ERROR:
-      return "internal error happened";
+    case E::SUCCESS:
```

Comment:
What is this for?

Reply:
When you print error message and it has code 0 it will write "unknown error", but what really happened is "success"